tests_integration/nativeapp/test_debug_mode.py

```diff
@@ -12,19 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import yaml
-import uuid
 from typing import Optional
 
-from snowflake.cli.api.project.util import generate_user_env
 from snowflake.connector.connection import SnowflakeConnection
 from snowflake.connector.errors import ProgrammingError
 
 from tests.project.fixtures import *
 from tests_integration.test_utils import pushd, enable_definition_v2_feature_flag
 
-USER_NAME = f"user_{uuid.uuid4().hex}"
-TEST_ENV = generate_user_env(USER_NAME)
+USER_NAME = os.environ.get("USER", "")
```

Comment:
this might fail on Windows. I suggest we make it deterministic, and pass a non-blank test user.
If you don't want to pass USER every time, we can default to some default_user value, and pass it as a fixture to the test:
`{project_name}_{default_user}{resource_suffix}`

Reply:
removed entirely, `USER` is now automatically passed by the CLI runner unless the test supplies its own